.github/workflows/docs/BuildAndPublish.md

```diff
@@ -0,0 +1,173 @@
+
+**CI/CD** (Continuous Integration and Continuous Deployment) refers to automating the process of building, testing, and publishing your game whenever you make updates. The idea is to speed up your game's release process so you can push updates frequently, to fix bugs quicker or add more game content.
+
+Using GitHub Actions, you can set up **workflows that automatically compile your Godot project** and **upload it to platforms** like itch.io whenever you tag a new release. This saves time, reduces manual errors, and helps keep your build and release process smooth and repeatable.
+
+> Note: You can set up all of this and still keep your game as a _Draft_ on itch.io. This is great for playtesting!
+
+## Prerequisites
+
+Before following this guide, make sure you have the following in place:
+
+- **GitHub Account & Repository:** Your Godot project should be pushed to a GitHub repository. If you haven’t yet, create one and upload your project files.
+- **Itch.io Account:** You’ll need an itch.io account to publish your game. Create one at itch.io if you don’t have it yet.
+- **Godot Export Presets Configured:** Ensure your export_presets.cfg file is properly configured in Godot for each platform you want to build (Windows, macOS, Linux, Web). The `build-and-publish.yml` will **trigger** these build config by name.
+
+## How to trigger the CICD?
+
+When you’re ready to publish a new version of your game, create a **GitHub release** tied to a version tag on the `main` branch. Using releases helps track updates, distribute builds, and communicate changes to players or testers.
+
+A new release will trigger the `build-and-publish.yml` workflow, which will **build your game** in the cloud and **publish it** to itch.io (if everything is setup).
+
+1. Ensure all desired changes are merged into the `main` branch. This is the version that'll get build and published.
+2. On Github, go to Release, then **draft a new release** ([here is a step by step guide](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release)). Create a new tag on you `main` branch using [semantic versioning](https://semver.org/). As a best practice, also prefix it with `v` in Github.
+   - `x.0.0` — Major Release. Large updates or milestones (e.g., new game systems, overhauled visuals, major gameplay changes). Example: `v1.0.0` for the full launch.
+   - `x.y.0` — Minor Update. New content or features that expand gameplay but remain backward-compatible. Example: `v1.1.0` for new levels or mechanics.
+   - `x.y.z` — Patch / Hotfix. Small updates, bug fixes, performance improvements, or balancing tweaks. Example: `v1.1.3` for fixing a crash or visual glitch.
+3. Publish the release. This will trigger the CICD in Github Action. Monitor its execution in the Actions tab on Github.
+
+Now, before creating a new Release, let's setup your game build and itch.io connection.
```

Comment:
I found it odd to jump into this section before finishing the setup. If the intent is to sell users on the eventual ease of use before going into setup, perhaps a short version of the first line (21) can be added to the introduction.